ui/Screen/SendFunds.svelte

```diff
@@ -0,0 +1,258 @@
+        placeholder="Select wallet you want to use"
+        bind:value={payer}
+        options={dropdownOptions} />
+      <div class="submit">
+        <Button on:click={() => nextStep()}>Review transfer</Button>
+      </div>
+    {/if}
+    {#if state === TransferState.Confirmation}
+      <header>
+        <div class="icon">
+          <Icon.ArrowUp style="fill: var(--color-primary)" />
+        </div>
+        <Title variant="big">Outgoing transfer</Title>
+        <div class="from-to">
+          <div class="from">
+            {#each dropdownOptions as option}
+              {#if option.value === $payerStore}
+                <Avatar
+                  size="small"
+                  title={option.avatarProps.title}
+                  variant={option.avatarProps.variant}
+                  avatarFallback={option.avatarProps.avatarFallback} />
+              {/if}
+            {/each}
+          </div>
+          <Icon.ArrowRight />
+          <div class="to">
+            <Title truncate style="color: var(--color-foreground-level-6);">
+              {$recipientStore}
+            </Title>
+          </div>
+        </div>
+      </header>
+      <Row variant="top" style="">
+        <div slot="left">
+          <Text
+            variant="regular"
+            style="color:var(--color-foreground-level-6);">
+            Amount
+          </Text>
+        </div>
+
+        <div slot="right">
+          <Rad rad={$amountStore} />
+        </div>
+      </Row>
+      <Row variant="middle" style="">
+        <div slot="left">
+          <Text
+            variant="regular"
+            style="color:var(--color-foreground-level-6);">
+            Transaction Fee
+          </Text>
+        </div>
+
+        <div slot="right">
+          <Rad rad={$amountStore} />
+        </div>
+      </Row>
+      <Row
+        variant="bottom"
+        style="border: 1px solid var(--color-foreground-level-2);">
+        <div slot="left">
+          <Title
+            variant="regular"
+            style="color:var(--color-foreground-level-6);">
+            Total
+          </Title>
+        </div>
+
+        <div slot="right">
+          <Rad rad={$amountStore} />
+        </div>
+      </Row>
+      <Row style="margin-top: 1.5rem;">
+        <div slot="left">
+          <Text
+            variant="regular"
+            style="color:var(--color-foreground-level-6);">
+            Funding source
+          </Text>
+        </div>
+
+        <div slot="right">
+          {#each dropdownOptions as option}
+            {#if option.value === $payerStore}
+              <Avatar
+                size="small"
+                title={option.avatarProps.title}
+                variant={option.avatarProps.variant}
+                avatarFallback={option.avatarProps.avatarFallback} />
+            {/if}
+          {/each}
+        </div>
+      </Row>
+      <div class="submit">
+        <Button variant="transparent" on:click={() => previousStep()}>
```

Comment:
```suggestion
        <Button variant="transparent" on:click={cancel}>
```

Reply:
Makes sense to name them more explicitly: https://github.com/radicle-dev/radicle-upstream/pull/712/commits/6b4d7fb0eb344641d5bd1f575446057ce0fc6ea4